tests/h/streamer/filter_test.py

```diff
@@ -1,13 +1,18 @@
 from datetime import datetime
+from random import random
```

Comment:
I forget - is the seed for the `random` function reproducible in the event of a test failure?

Reply:
No, but we don't actually rely on the output at all. This is just to make sure the speed check isn't massively cheating by actually only having one value over and over. There were some implementations of this where it would suddenly (and wrongly) look like it was 1000x times faster or something because we had such unrepresentative data.